### DISCUSSION TITLE
Add "Hit Judgement Colour" to `ArgonSongProgress`

### DISCUSSION THREAD

**Discussion by ikkue** [ppy/osu#22453] [Ideas] [answered]
Similar to that in osu!stream where the progress bar's colour appears green in the sections the player received an `OK`, yellow for `MEH`, and red for `MISS`. Ideally, the colours are the same used in the result screen and `JudgementCounterDisplay`.

https://user-images.githubusercontent.com/11213135/215296966-49ee60fa-eb1b-4292-a1c3-c6b2848c62d0.mp4

This could be an alternative option to the "difficulty graph" in the skin editor, or they could be two separate toggles entirely. (The latter would probably mean a different design/implementation for the "difficulty graph" as opposed to the current "different shades of gray", as toggling both would likely result in an overlap of the two colour schemes).

**Comment by Joehuu** [comment, marked as answer]
https://github.com/ppy/osu/issues/5681
